### PR TITLE
Update `cookbooks/new_ping documentation` for BigQuery ingestion

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This updates the cookbook for adding a new ping to reference the new ping tables in BigQuery. This removed references to the old AWS pipeline, including parquet schemas and the CEP.

### CODE REVIEW

**Comment by tdsmith** [comment]
I was about to edit this page to add some content like this at the top; would you mind rolling it in here?

> Most new data collection in Firefox does not require creating a new ping document type. To add a histogram, scalar, or event collection to Firefox, please see the documentation on [adding a new probe](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/start/adding-a-new-probe.html).

**Comment by acmiyaguchi** [comment]
@tdsmith Sure thing, would it make sense to move the origin of this PR so you can push your changes to this branch too?

File: src/cookbooks/new_ping.md [hunk 1]
```diff
@@ -3,189 +3,247 @@
...
+where the data falls under the [Data Collection
+Categories](https://wiki.mozilla.org/Firefox/Data_Collection).
+
+## Submit Schema to `mozilla-services/mozilla-pipeline-schemas`
+
+Create a PR including a template and rendered schema to `mozilla-pipeline-schemas`. Add at least one
+validation ping that exercises the structure of schema as a test. These pings are validated during
+the build and help catch mistakes during the writing process.
+
+### Example: A rendered schema for response times
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: Nice!

File: src/cookbooks/new_ping.md [hunk 2]
```diff
@@ -3,189 +3,247 @@
 Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide
 to find out.
 
-Write Your Questions
---------------------
+Most new data collection in Firefox does not require creating a new ping document type. To add a
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: Add something in bold before this: **Note** or similar

File: src/cookbooks/new_ping.md [hunk 3]
```diff
@@ -3,189 +3,247 @@
...
 specific, concrete asks that can be translated to data points. This will also make it easier down
 the line as you start data review.
 
-More detail on how to design and implement new pings for Firefox Desktop [can be found here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html).
+More detail on how to design and implement new pings for Firefox Desktop [can be found
+here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html).
+
+## Choose a Namespace and DocType
+
+For new telemetry pings, the namespace is simply `telemetry`. For non-Telemetry pings, choose a
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: I believe this needs more scoping now. IMHO we shouldn't have any more custom pings come in from the `telemetry` namespace, except those added by the Firefox Client Telemetry team.

File: src/cookbooks/new_ping.md [hunk 4]
```diff
@@ -3,189 +3,247 @@
...
-Both namespace and DocType are limited to the pattern `[a-zA-Z-]`. In other words, hyphens and letters from the [ISO basic Latin alphabet](https://en.wikipedia.org/wiki/ISO_basic_Latin_alphabet).
+## Create a Schema
 
-Create a Schema
----------------
-Use JSON Schema to start with. See the ["Adding a new schema" documentation](https://github.com/mozilla-services/mozilla-pipeline-schemas#adding-a-new-schema) and examples schemas in the
-[Mozilla Pipeline Schemas repo](https://github.com/mozilla-services/mozilla-pipeline-schemas/).
-This schema is just used to validate the incoming data; any ping that doesn't match the schema
-will be removed. Validate your JSON Schema using a
+Use JSON Schema to start with. See the ["Adding a new schema"
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: JSON schema is now the only option!

Comment (fbertsch) [reply src/cookbooks/new_ping.md]: Let's also be explicit here that you no longer need parquet schemas.

File: src/cookbooks/new_ping.md [hunk 5]
```diff
@@ -3,189 +3,247 @@
...
-And for non-Telemetry data:
-- `geoCountry`
-- `geoCity`
-- `geoSubdivision1`
-- `geoSubdivision2`
-- `documentId`
-
-*Important Note*: Schema evolution of nested structs is currently broken, so you will not be able to add
-any fields in the future to your `metadata` section. We recommend adding any that may seem useful.
+Ensuring the ping contains a top-level `id` will enable document-level deduplication, which catches
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: a _unique_ top-level `id`

File: src/cookbooks/new_ping.md (hunk 2 above)
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: We should note here also that none of this is required for Glean.

File: src/cookbooks/new_ping.md [hunk 6]
```diff
@@ -3,189 +3,247 @@
...
+the following [Airflow
+DAG](https://github.com/mozilla/telemetry-airflow/blob/master/dags/probe_scraper.py). The latest
+schemas can be viewed at
+[`mozilla-pipeline-schemas/generated-schemas`](https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/generated-schemas).
+
+## Start Sending Data
+
+Use the built-in Telemetry APIs when possible. A few examples are the [Gecko Telemetry
+APIs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html),
+the [Android Telemetry APIs](https://github.com/mozilla-mobile/telemetry-android), or the [iOS
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: I sort of think we shouldn't even include Android Telemetry, and note again that Android should use Glean, which doesn't require this.

File: src/cookbooks/new_ping.md [hunk 7]
```diff
@@ -3,189 +3,247 @@
...
+SELECT
+    count(*) as n_rows
+FROM
+  `moz-fx-data-shared-prod.telemetry_live.main_v4`
+WHERE
+  submission_timestamp > TIMESTAMP_SUB(current_timestamp, INTERVAL 30 minute)
+```
+
+The second table that is created is the clustered table under
+`moz-fx-data-shared-prod.<namespace>_stable.<doctype>_v<docversion>`. This table will only contain
```
Comment (fbertsch) [inline src/cookbooks/new_ping.md]: I'm thinking we leave `_stable` out and have people just query the views.

**Pull request merged**